mmdeploy/codebase/mmedit/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from .contextual_attention import *  # noqa: F401,F403
```

Comment:
use `from . import contextual_attention` would be better. Hiding the rewriter to avoid name conflict.